scripts/west_commands/runners/esp32.py

```diff
@@ -53,8 +53,8 @@ def do_add_parser(cls, parser):
                             help='partition table load address')
         parser.add_argument('--esp-app-address', default='0x10000',
                             help='application load address')
-        parser.add_argument('--esp-device', default='/dev/ttyUSB0',
-                            help='serial port to flash, default /dev/ttyUSB0')
+        parser.add_argument('--esp-device', default=os.environ.get('ESPTOOL_PORT', None),
+                            help='serial port to flash')
```

Comment:
Why not keep the /dev/ttyUSB0 as a last default?
 
```
        parser.add_argument('--esp-device', default=os.environ.get('ESPTOOL_PORT', '/dev/ttyUSB0'),
                            help='serial port to flash')
```

Reply:
@b0661, `esptool.py` will loop over the available serial ports and use `/dev/ttyUSB0` if your ESP board is in that port. This update in `esp32.py` make sure of that. Also, exporting `ESPTOOL_PORT=/dev/ttyUSB0` allows making a default port as well.